geode-core/src/distributedTest/java/org/apache/geode/internal/cache/execute/FunctionRetryTestBase.java

```diff
@@ -264,7 +264,7 @@ private Execution getExecutionTarget(ExecutionTarget executionTarget) {
         break;
       case SERVERS_REGION_SERVICE:
         execution = FunctionService
-            .onServer(clusterStartupRule.getClientCache().getRegion(regionName).getRegionService())
+            .onServers(clusterStartupRule.getClientCache().getRegion(regionName).getRegionService())
```

Comment:
Is this an intentional change?

Reply:
@pivotal-jbarrett Yes, while running tests I noticed that it was not calling onServers() which its supposed to be.